base/dates/parse.jl

```diff
@@ -125,9 +125,9 @@ Returns a 2-element tuple `(values, pos)`:
   the passed in type.
 * `pos::Int`: The character index at which parsing stopped.
 """
-@generated function tryparsenext_internal{T<:TimeType}(
+@generated function tryparsenext_internal(
     ::Type{T}, str::AbstractString, pos::Int, len::Int, df::DateFormat, raise::Bool=false,
-)
+) where T<:TimeType
```

Comment:
Not `@tkelman`, but agreed re. the linebreaking being odd. Perhaps collapse the signature as you deem consistent with the rest of the codebase, e.g. to two lines with appropriate indentation of the second line?